website/docs/plugin/framework/migrating/attributes-blocks/types.mdx

```diff
@@ -0,0 +1,202 @@
+---
+page_title: 'Attribute Types: Migrating from SDKv2 to the Framework'
+description: >-
+  Migrate attribute type from SDKv2 to the plugin Framework
+---
+
+# Attribute Types
+
+An attribute either contains a primitive type, such as an integer or a string, or contains other attributes. Attributes
+that contain other attributes are referred to as nested attributes, and are implemented by populating the
+`NestedAttributes` field on the `tfsdk.Attribute` struct. Refer to
+[Schemas - Attributes](/plugin/framework/schemas#type) in the Framework documentation for details.
+
+This page explains how to migrate a primitive attribute from SDKv2 to the plugin Framework. For an example of
+migrating a nested block to a nested attribute, refer to [Provider Schema](/plugin/framework/migrating/providers#example-1) in the Framework documentation.
```

Comment:
```suggestion
migrating a nested block to a nested attribute, refer to [Provider Schema](/plugin/framework/migrating/providers#example-1) in the Framework documentation.
```
I think this is the wrong URL? Also we should say "in this guide." instead of "in the Framework documentation." if we're linking to within the guide.

Reply:
The link is correct, but the text for the link should now appear as `Providers` following the update where we moved `/providers/schema.mdx` => `/providers/index.mdx`. I believe this is the only example we have in the migration guide which shows migration from a nested block to a nested attribute. I've updated as follows:

> migrating a nested block to a nested attribute, refer to [Providers](/plugin/framework/migrating/providers#example-1) in this guide.